### DISCUSSION TITLE
[Question]: Region Mapper available for this?

### DISCUSSION THREAD

**Discussion by Woisek** [hako-mikan/sd-webui-regional-prompter#128] [Q&A]
Just like this for Latent Couple -> https://github.com/CodeZombie/latentcoupleregionmapper/, is there a similar helper tool for this regional prompter somewhere? 🤔
Or maybe someone who want to create such tool for us? 😃

**Comment by Symbiomatrix** [comment]
Nothing that polished, but there's inpaint+. Works in native gradio and the regions can be drawn freehand or uploaded.
Plus, it only picks specific colours from the uploaded file, so theoretically you could upload a tracing grid (or a more general purpose layout image).
Given the limitations imposed by the gui and my (lack of) skill, I think it does the job relatively well.
https://github.com/hako-mikan/sd-webui-regional-prompter#mask-regions-aka-inpaint-experimental-function

**Reply by Woisek** [reply]
Thanks for the tip. I will look into it. 👍
